cmd/jujud/agent/machine.go

```diff
@@ -1106,14 +1106,24 @@ func (a *MachineAgent) startModelWorkers(cfg modelworkermanager.NewModelConfig)
 		logger.Errorf("unable to configure db logging for model: %v", err)
 	}
 
+	charmRevisionUpdateInterval := 24 * time.Hour
```

Comment:
As we get more of these, it'll make the function unwieldy, perhaps extract into a function?

Reply:
yup